test/acceptance/html.js

```diff
@@ -24,6 +24,40 @@ let suite = {
       })()
     },
     {
+      description: 'HTML comment with text inside',
+      template: '<!-- hi there -->',
+      context: {},
+      expectedDom: (() => {
+        let frag = document.createDocumentFragment();
+        let comment = document.createComment(' hi there ');
+        frag.appendChild(comment);
+        return frag;
+      })()
+    },
+    {
+      description: 'HTML comment with HTML inside',
+      template: '<!-- <script>alert(\'bad stuff!\');</script> -->',
+      context: {},
+      expectedDom: (() => {
+        let frag = document.createDocumentFragment();
+        let comment = document.createComment(' <script>alert(\'bad stuff!\');</script> ');
+        frag.appendChild(comment);
+        return frag;
+      })()
+    },
+    {
+      description: 'HTML comment within HTML attribute',
+      template: '<div class="<!-- comment -->"></div>',
```

Comment:
Right, and I want to make sure that it is not interpreted as a comment. Maybe the test description needs to be updated?